app/models/testimonial.rb

```diff
@@ -0,0 +1,3 @@
+class Testimonial < ApplicationRecord
+  belongs_to :case_study
```

Comment:
@NRaeewal I think, we need to add optional: true, because some of the testimonials will be without case studies.